sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/v2/state/StateDataSourceTransformWithStateSuite.scala

```diff
@@ -563,6 +673,58 @@ class StateDataSourceTransformWithStateSuite extends StateStoreMetricsTest
     }
   }
 
+  testWithChangelogCheckpointingEnabled("state data source cdf integration -" +
```

Comment:
nit: space after `-`, to be consistent with above test name

Reply:
Done